### PR TITLE
Remove trivial ecma_number arithmetic functions

### PR DESCRIPTION
**Pull request by rerobika**
The affected function calls have been replaced with the appropriate arithmetic operands.

JerryScript-DCO-1.0-Signed-off-by: Robert Fancsik frobert@inf.u-szeged.hu

### CODE REVIEW

**Comment by LaszloLango** [comment]
I don't see any improvement here. This is just a noise. I think these inlining are done by compiler automatically, because the binary size did not increase.

**Comment by rerobika** [comment]
@LaszloLango I've updated the result with the run time standard deviation and highlighted the tests and results where the gain is magnitude greater then the std deviation. That's why I think it's more then noise.

**Comment by rerobika** [comment]
Also I've compared the two binaries (with patch and without patch) and there is difference between them. It's means inlining these functions has real effect.

File: jerry-core/ecma/base/ecma-helpers-number.c [hunk 1]
```diff
@@ -687,7 +687,7 @@ ecma_number_calc_remainder (ecma_number_t left_num, /**< left operand */
  *
  * @return number - result of addition.
  */
-ecma_number_t
+inline ecma_number_t __attr_always_inline___
```
Comment (zherczeg) [inline jerry-core/ecma/base/ecma-helpers-number.c]: Please just remove these functions.

**Comment by rerobika** [comment]
@zherczeg Your suggestion have been applied!

File: jerry-core/vm/vm.c [hunk 2]
```diff
@@ -1696,8 +1696,8 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           if (ecma_is_value_float_number (left_value)
               && ecma_is_value_number (right_value))
           {
-            ecma_number_t new_value = ecma_number_add (ecma_get_float_from_value (left_value),
-                                                       ecma_get_number_from_value (right_value));
+            ecma_number_t new_value = ecma_get_float_from_value (left_value) +
+            ecma_get_number_from_value (right_value);
```
Comment (zherczeg) [inline jerry-core/vm/vm.c]: This is not nice alignment. What is the difference between float value and number value?

Comment (rerobika) [reply jerry-core/vm/vm.c]: Number value not specifies that the stored value is float or integer. Unfortunately it is not a nice alignment, but this is what vera++ requires.

Comment (zherczeg) [reply jerry-core/vm/vm.c]: I think you can use `()` brackets to help aligning

Comment (rerobika) [reply jerry-core/vm/vm.c]: It worked! Thanks for your advice!

**Pull request merged**